Carbon/calculations/travel_calculations/calcPublic.js

```diff
@@ -0,0 +1,18 @@
+/*
+    Takes the amount of miles traveled by bus or train and returns the carbon produced
+    Paramater
+    miles - the miles taken on public transprot
+    Return
+    CO2 produced in lbs
+
+*/
+const calcPublic = (miles, type_of_transport) => {
+    if(type_of_transport == "bus")
+    {
+        return Math.round(miles * 0.659182);
+    }
+    return Math.round(miles * 0.390218);
```

Comment:
Explicitly write an if statement for "train" so no other values can pass in.

Reply:
What should it return if nothing is passed in then?

Reply:
Error catching... basically an error on dev side, not user side :)